data/src/main/java/net/androidweekly/data/repositories/issues/IssuesRepositoryImpl.kt

```diff
@@ -26,8 +26,19 @@ class IssuesRepositoryImpl(
         return localIssuesDao.getLatestIssues()
     }
 
-    override suspend fun getPastIssues(): List<Issue>? {
-        return remoteIssuesDao.getAllIssues().channel?.items
+    override suspend fun getRemotePastIssues(): List<Issue>? {
+        val items = remoteIssuesDao.getAllIssues()
+            .channel?.items
+
+        items?.forEach {
```

Comment:
Why not create another method in `localIssuesDao` to insert a list instead of forEach and insert item by item?